### PR TITLE
HPCC-13579 Correct doc error MonitorLogicalFileName

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@jakesmith Please review

### CODE REVIEW

File: docs/ECLStandardLibraryReference/SLR-Mods/MonitorLogicalFileName.xml [hunk 1]
```diff
@@ -98,6 +97,10 @@
   <emphasis>event</emphasis> with the name of the triggering object as the
   event subtype (see the EVENT function).</para>
 
+  <para>This function does not support wildcard characters. To monitor for
+  files using wildcards, use the <link
```
Comment (jakesmith) [inline docs/ECLStandardLibraryReference/SLR-Mods/MonitorLogicalFileName.xml]: @JamesDeFabia 
Is it just me, or would that sound better without the 'for' ?
i.e. "To monitor files using wildcards .. "

Comment (jakesmith) [inline docs/ECLStandardLibraryReference/SLR-Mods/MonitorLogicalFileName.xml]: Also, this makes it sound like MonitorFile can be used for logical files, whereas MonitorFile is just for monitoring physical files/directories.

**Comment by JamesDeFabia** [comment]
@jakesmith Edits made based on your feedback. Please have another look

**Comment by jakesmith** [comment]
Looks good
@richardkchapman - ready to merge

**Pull request merged**